plugins/lookup/dig.py

```diff
@@ -42,6 +42,11 @@
         default: false
         type: bool
         version_added: 3.6.0
+      fail_on_error:
+        description: Abort execution on lookup errors.
```

Comment:
How about making it more clear that this is something you really want to set to `true`:
```suggestion
        description:
          - Abort execution on lookup errors.
          - The default for this option will likely change to C(true) in the future. The current default,
            C(false), is used for backwards compatibility, and will result in empty strings or the string
            C(NXDOMAIN) in the result in case of errors.
```